DSharpPlus/Entities/Channel/Overwrite/DiscordOverwriteBuilder.cs

```diff
@@ -29,13 +29,24 @@ public sealed class DiscordOverwriteBuilder
         public SnowflakeObject Target { get; private set; }
 
         /// <summary>
-        /// Creates a new Discord permission overwrite builder. This class can be used to construct permission overwrites for guild channels, used when creating channels.
+        /// Creates a new Discord permission overwrite builder for a member. This class can be used to construct permission overwrites for guild channels, used when creating channels.
         /// </summary>
-        public DiscordOverwriteBuilder()
```

Comment:
If you want this to be merged in the patch release, you need to add the `Obsolete` attribute rather than removing it. Make sure to set the second parameter to `false` as well, so it doesn't throw an error.